ui/.storybook/main.ts

```diff
@@ -23,7 +23,6 @@ const config: StorybookConfig = {
   },
   docs: {
     autodocs: 'tag',
-    docsMode: true,
```

Comment:
AFAICT, the only difference is that stories get their own sub menu items in the nav. Doesn't seem too cluttered since you can collapse

<img width="196" alt="image" src="https://github.com/inngest/inngest/assets/20100586/3f173c38-3012-46fd-91f9-65423f406859">